core/src/main/java/com/uber/m3/tally/ScopeKey.java

```diff
@@ -0,0 +1,36 @@
+package com.uber.m3.tally;
+
```

Comment:
afaik CI pipeline is broken, could you please run the tests and check style locally and fix the errors.
```
./gradlew checkstyleMain
```